tools/ccm-service-kit/product-mapping/service_compare.csv

```diff
@@ -0,0 +1,3065 @@
+product,service_type,service_category,cloud_provider
```

Comment:
Just out of curiosity, what is this file for?

Reply:
It is for the product mappings. It is just an example of the needed table.